frontend/src/components/Input/style.tsx

```diff
@@ -1,14 +1,22 @@
 import { styled } from 'styled-components';
 
+export type InputWidth = 'small' | 'medium' | 'fill';
+
 type StyledInputProps = {
-  $width?: number;
+  $width?: InputWidth;
   $center?: boolean;
   disabled?: boolean;
   $required?: boolean;
 };
 
+const TYPE: Record<string, string> = {
+  small: '410px',
+  medium: '550px',
+  fill: '100%',
+};
```

Comment:
추가로 TYPE이라는 이름이 범용적인 것 같아요
```suggestion
const TYPE: Record<InputWidth, string> = {
  small: '410px',
  medium: '550px',
  fill: '100%',
};
```

Reply:
`SIZE`로 변경했습니다🥲

[fba642f](https://github.com/woowacourse-teams/2023-stamp-crush/pull/692/commits/fba642f3ac2262dc75b1d8df20252e752a6f70ce)

Reply:
타입보단 낫네용